src/coreclr/vm/threads.h

```diff
@@ -706,11 +702,10 @@ class Thread
                                                       // at the beginning of wait.
         // unused                       = 0x00040000,
         // unused                       = 0x00080000,
-        TSNC_RaiseUnloadEvent           = 0x00100000, // Finalize thread is raising managed unload event which
-                                                      // may call AppDomain.Unload.
+        // unused                       = 0x00100000,
```

Comment:
BTW: `TSNC_CLRCreatedThread`, `TSNC_RaiseUnloadEvent` and `TSNC_InsideSyncContextWait` look unused as well.